tests/try_scraper/test_invalid_revision.py

```diff
@@ -5,24 +5,18 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from mock import patch
-
+import pytest
 from mozdownload import TryScraper
 import mozdownload.errors as errors
-import mozhttpd_base_test as mhttpd
-
 
-class TestTryScraperInvalidParameters(mhttpd.MozHttpdBaseTest):
-    """test mozdownload TryScraper class with invalid parameters"""
 
-    @patch('mozdownload.treeherder.Treeherder.query_builds_by_revision')
-    def test_scraper(self, query_builds_by_revision):
-        """Testing download scenarios with invalid parameters for TryScraper"""
-        query_builds_by_revision.return_value = []
+@patch('mozdownload.treeherder.Treeherder.query_builds_by_revision')
+def test_scraper(query_builds_by_revision, httpd, tmpdir):
+    """Testing download scenarios with invalid parameters for TryScraper"""
+    query_builds_by_revision.return_value = []
 
-        with self.assertRaises(errors.NotFoundError):
-            TryScraper(destination=self.temp_dir,
-                       base_url=self.wdir,
-                       logger=self.logger,
-                       platform='win32',
-                       revision='abc'
-                       )
+    with pytest.raises(errors.NotFoundError):
+        assert TryScraper(destination=tmpdir,
```

Comment:
What are you trying to assert here?

Reply:
This verifies whether using the provided parameters TryScraper can be instantiated or not. Try Scrapper internally calls Scraper.__init__ which checks the provided parameters. So for example in `platform` if i provide `win32random` mistakenly then the test will fail.

Reply:
Sure, but I don't see the need for an assert here. If it fails `__init__` will raise an exception, which then bubbles up to the test itself, because `pytest.raises` only checks for `NotFoundError`. The assert won't be run at any time.

Reply:
I agree, I will remove `assert` from here. Moreover, I was also wondering about the type of arguments we pass to any instantiation should there be any checks whether the type is the one which we expect.

Reply:
Good point. Those tests we don't have yet. So if you have something in mind feel free to create those in a separate test file.